Model/Capabilities.php

```diff
@@ -129,6 +129,17 @@ public function getTokenizationMethods()
         return $this->capabilitiesAPI->getTokenizationMethods();
     }
 
+    /**
+     * @return int
```

Comment:
What if the installment min amount provided by capability API doesn't end with `00`?